tests/test_parse.py

```diff
@@ -114,18 +113,22 @@ class MyModel(BaseModel):
     assert isinstance(m.__root__.__root__[0], str)
 
 
-@pytest.mark.xfail(reason='working on V2')
 def test_json():
-    assert Model.parse_raw('{"a": 12, "b": 8}') == Model(a=12, b=8)
+    # TODO: Are we planning to keep model_validate_json?
+    assert Model.model_validate_json('{"a": 12, "b": 8}') == Model(a=12, b=8)
 
 
 @pytest.mark.xfail(reason='working on V2')
 def test_json_ct():
+    # TODO: Should this test be dropped now that parse_raw is gone?
```

Comment:
Question 7: This seemed to be checking specifically for the behavior of the `content_type` arg; since `parse_raw` is gone and `model_validate_json` doesn't have this arg, can we drop this test completely?

Reply:
yes, drop test.